gem/lib/pupperware/spec_helper.rb

```diff
@@ -648,7 +648,7 @@ def orchestrate_puppet_run(
         pe_console_services: 'pe-console-services.test',
         pe_orchestration_services: 'pe-orchestration-services.test'
       )
-    image = 'artifactory.delivery.puppetlabs.net/pe-and-platform/pe-client-tools:kearney-latest'
+    image = 'artifactory.delivery.puppetlabs.net/pe-and-platform/pe-client-tools:latest'
```

Comment:
Should this image be a parameter with a default value?

Reply:
Seems like we should make this gem be version-agnostic so we don't have to get into the business of versioning the gem.

Reply:
Yeah, makes sense. Let me take a look at that.

There are a few other changes I want to make to some helpers to remove `.test` as well... so I'll try and make those together.